openms-wiki/contributors-quickstart-guide.md

```diff
@@ -0,0 +1,61 @@
+# Contribute to OpenMS
+
+If you would like to contribute to OpenMS:
+
+* Familiarise yourself with our [online documentation](https://abibuilder.informatik.uni-tuebingen.de/archive/openms/Documentation/release/latest/html/index.html).
+
+* Learn how to [build OpenMS](https://github.com/OpenMS/OpenMS/wiki/Building-OpenMS).
+
+* Start reading the [OpenMS tutorial for developers](https://abibuilder.informatik.uni-tuebingen.de/archive/openms/Documentation/release/latest/html/OpenMS_tutorial.html).
+
+Any questions can be directed at the mailing list.
+
+## Technical Documentation
+Untested installers, containers, etc., known as the nightly snapshot, are released every night. They generally pass automated continuous integration tests but no manual tests.  
+Access the documentation for the nightly snapshot of [OpenMS develop branch](https://github.com/OpenMS/OpenMS/tree/develop) [here](https://abibuilder.informatik.uni-tuebingen.de/archive/openms/Documentation/nightly/html/index.html). Access the doxygen log from [here](https://abibuilder.informatik.uni-tuebingen.de/jenkins/job/openms_nightly_packaging/lastBuild/compiler=appleclang-7.3.0,os_label=elcapitan/artifact/build/doc/doxygen/doxygen-error.log).
```

Comment:
instead of:
> Access the documentation for the nightly snapshot of

let's say
The documentation for the nigtly snapshot .... can be found online [here]().